src/main/java/io/rocketpartners/cloud/action/sql/SqlDb.java

```diff
@@ -147,6 +148,22 @@ public String getType()
       return null;
    }
 
+   public void truncateDb()
```

Comment:
This is not a truncate action.  It deletes any db files that existed from previous executions of the test case.